cmd/ratchet/main.go

```diff
@@ -108,16 +109,28 @@ func main() {
 		log.Fatalf("error setting up ingestion worker: %v", err)
 	}
 
+	// Report worker setup
+	reportWorker, err := report_worker.New(db, slackIntegration.SlackClient())
+	if err != nil {
+		log.Fatalf("error setting up report worker: %v", err)
+	}
+
 	// Background job setup
 	workers := river.NewWorkers()
 	river.AddWorker(workers, classifier)
 	river.AddWorker(workers, ingestionWorker)
+	river.AddWorker(workers, reportWorker)
 	riverClient, err := background.New(db, workers)
 	if err != nil {
 		log.Fatalf("error setting up background worker: %v", err)
 	}
 	bot.RiverClient = riverClient
 
+	// Initialize bot (this will set up periodic jobs)
+	if err := bot.Initialize(); err != nil {
+		log.Fatalf("Failed to initialize bot: %v", err)
+	}
+
```

Comment:
We probably want to register this periodic job as part of `AddChannel`. And if a channel sees no activity in report period, do nothing (since we don't currently remove entries from `channels` table if bot is removed from that channel).

Reply:
yeah, that's correct, the initial insert should happen in `AddChannel`, I'll fix it.